main.py

```diff
@@ -603,6 +603,15 @@ def handle_flows_install_delete(self, event):
             return
 
         force = bool(event.content.get("force", False))
+        if not flow_dict["flows"]:
```

Comment:
This will result in `KeyEror` is `flows` isn't set. It's covering this validation too.

Reply:
I got confused `validate_cookies_and_masks` should be accepting flow list and not flows dictionary. Added extraction for `flow_dict['flows']` and catching exception.